### ISSUE TITLE
Laravel version 7.6.2 cannot installl, The dependency conflicts

### ISSUE THREAD

**Issue by wilbur-oo** [php-pm/php-pm-httpkernel#168]
artisan -V --ansi
Laravel Framework 7.6.2

composer why-not php-pm/php-pm 2.0.3 -t -r
php-pm/php-pm 2.0.2 PHP-PM is a process manager, supercharger and load balancer for PHP applications.
├──php-pm/php-pm 2.0.3 (requires symfony/error-handler ^4.4) (circular dependency aborted here)
└──laravel/laravel (does not require symfony/error-handler but v5.0.7 is installed) (circular dependency aborted here)

composer require php-pm/httpkernel-adapter -vvv
- Can only install one of: symfony/event-dispatcher[4.4.x-dev, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.0, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.0-BETA1, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.0-BETA2, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.0-RC1, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.1, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.2, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.3, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.4, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.5, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.6, v5.0.7].
    - Can only install one of: symfony/event-dispatcher[v4.4.7, v5.0.7].
    - Installation request for symfony/event-dispatcher (locked at v5.0.7) -> satisfiable by symfony/event-dispatcher[v5.0.7].

**Comment by andig** [comment]
What is your install command? Are you upgrading dependencies?

**Comment by wilbur-oo** [comment]
> What is your install command? Are you upgrading dependencies?

I am planning to use your project in Laravel, but there was a dependency conflict when installing the `php-pm/httpkernel-adapter` package.
I first introduced the `composer require php-pm / php-pm package` in the project, and then reported an error when introducing `composer require php-pm / httpkernel-adapter`

**Comment by andig** [comment]
Ok, but could you answer the question? What is your (verbatim) install command?

Viele Grüße,
Andreas

**Comment by wilbur-oo** [comment]
I don't understand what your mean by the installation command, do you mean composer?
Let me talk about my installation process,
1.composer create-project laravel/laravel lv-pm
2.composer require php-pm/php-pm
3.composer require php-pm/httpkernel-adapter
Then I got an error.

**Comment by andig** [comment]
Exactly. Could you please try- as suggested- to run composer with `--update-with-dependencies`?

**Comment by wilbur-oo** [comment]
@andig

**Comment by wilbur-oo** [comment]
@andig I think I found the cause of the dependency conflict, but I have n’t determined which package caused it. Using the --update-with-dependencies option you mentioned in the current directory did not work. But I ’m working on a new laravel In the new pm project, the package was successfully introduced. So it should be the problem caused by the introduction of other packages. I will feedback after I am sure, thank you for your reply.

**Comment by andig** [comment]
You could also try `--update-with-all-dependencies`, `symfony/error-handler ^5` is definitely allowed.

**Comment by wilbur-oo** [comment]
> You could also try `--update-with-all-dependencies`, `symfony/error-handler ^5` is definitely allowed.

Yes, I am already doing this. Thanks for your suggestions.

**Comment by andig** [comment]
Did you solve this or could provide steps to reproduce from the point of installing Laravel on?
